src/entities/Votes/utils.ts

```diff
@@ -163,3 +182,49 @@ export function abbreviateNumber(vp: number) {
 function getFloorOrZero(number?: number) {
   return Math.floor(number || 0)
 }
+
+export function getVoteSegmentation(votes: Record<string, Vote> | null | undefined): VoteSegmentation<Vote> {
+  const highQualityVotes: Record<string, Vote> = {}
```

Comment:
Maybe this `Record<string, Vote>` type could have a name that can help us name the functions return this type 🤔